engine/security/trust/index.md

```diff
@@ -97,13 +97,15 @@ The following image depicts the various signing keys and their relationships:
 
 > **WARNING**
 >
-> The root key once lost is not recoverable.
-> Correcting this loss requires intervention from [Docker
-> Support](/support/) to reset the repository state. This loss
-> also requires **manual intervention** from every consumer that used a signed
-> tag from this repository prior to the loss.
+>The root key once lost is not recoverable. If you lose any other key, send an email to [Docker Hub Support](mailto:hub-support@docker.com). This loss also requires manual intervention from every
+consumer that used a signed tag from this repository prior to the loss.
 {:.warning}
 
+
+
+
+
+
```

Comment:
There are a lot of empty lines here, it's best if we only have two newlines per block. Some markdown renderers would render empty paragraphs here which would create a big margin.

Reply:
I'm surprised the linter didn't get this. Isn't this being check there?

Reply:
Addressed in commit 3.